spek-ide-plugin/intellij-common/build.gradle

```diff
@@ -0,0 +1,11 @@
+apply from: "$rootDir/gradle/common/dependencies.gradle"
+apply from: "$rootDir/gradle/common/idea-plugin.gradle"
+
+intellij {
+    version = '2016.3'
```

Comment:
Seems like a pretty old version. Should we update it somehow?

Reply:
nah, this is good. This is the minimum version we support.